src/drivers/mct/bld/testdir/env_build.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<file id="env_build.xml">
+  <header>
+    These variables SHOULD NOT be changed once the model has been built.
+    urrently, these variables are not cached.
+    Note1: users SHOULD NOT modify BUILD_COMPETE below
+    this is done automatically by the scripts.
+    </header>
```

Comment:
Suspicious: seeing many new files.

Reply:
This entire directory "src/drivers/mct/bld" is not present in ESMCI cime.